grafana-operator/grafana-operator
# feat: whitelist log.frontend config

## Description

Fixes #474 by whitelisting the `log.frontend` config options.

## Type of change




## Verification steps

1. Deploy Grafana
2. Add this to the spec of the CR:

```
config:
  log.frontend:
    enabled: true
```

3. Verify that the `log.frontend` option is reflected in grafana.ini
